doc.go

```diff
@@ -2,9 +2,7 @@
 // Licensed under the LGPLv3, see LICENCE file for details.
 
 /*
-[godoc-link-here]
```

Comment:
This is used by the Makefile to inject a godoc.org badge. The Makefile should probably just prepend the badge to the file instead, rather than requiring it to be in the godoc in the first place.

Reply:
Sorry this took so long, somehow I missed this PR and then `godoc2md` was not running for me with newer Go versions (https://github.com/davecheney/godoc2md/pull/27) but I managed to make it work. I made the Makefile inject the badge instead of search and replace.